gtnh-modpack.json

```diff
@@ -720,6 +744,18 @@
       "side": "BOTH",
       "maven": "http://jenkins.usrv.eu:8081/nexus/content/repositories/releases/com/github/GTNewHorizons/HoloInventory/"
     },
+    {
+      "name": "HungerOverhaul",
```

Comment:
This is for pack building and we can't distribute this mod.  If you want to include this for dependencies/compiling adjust "side" to NONE